vite.config.ts

```diff
@@ -22,15 +27,35 @@ export default defineConfig({
 						'**/*.css',
 						'**/*.test.{js,jsx,ts,tsx}',
 						'**/__*.*',
-						// This is for server-side utilities you want to colocate next to
-						// your routes without making an additional directory.
-						// If you need a route that includes "server" or "client" in the
-						// filename, use the escape brackets like: my-route.[server].tsx
+						// This is for server-side utilities you want to colocate
+						// next to your routes without making an additional
+						// directory. If you need a route that includes "server" or
+						// "client" in the filename, use the escape brackets like:
+						// my-route.[server].tsx
 						'**/*.server.*',
 						'**/*.client.*',
 					],
 				})
 			},
 		}),
+		process.env.SENTRY_AUTH_TOKEN ??
+			sentryVitePlugin({
+				disable: MODE !== 'production',
+				authToken: process.env.SENTRY_AUTH_TOKEN,
+				org: process.env.SENTRY_ORG,
+				project: process.env.SENTRY_PROJECT,
+				release: {
+					name: process.env.COMMIT_SHA,
+					setCommits: {
+						auto: true,
+					},
+				},
+				sourcemaps: {
+					filesToDeleteAfterUpload: await glob([
+						'./public/**/*.map',
```

Comment:
Are there any files that match this glob?

Reply:
so, when I ssh into my own app project, I don't see that the `public` folder has any `.map`s in it. But I did notice that `.server-build` does, so I've replaced the dirs. What do you think ?